src/test/bench_test/benchmark.cpp

```diff
@@ -191,6 +235,58 @@ void benchmark::read_random(thread_arg *thread)
     thread->stats.add_message(msg);
 }
 
+void benchmark::multi_get_random(thread_arg *thread)
```

Comment:
Can we set the distribution of the dataset?

Reply:
Do you means the count of values for the same hashkey? 
To reuse params nums for other tests, I gave it a default number of 100. Refactoring is complex, and I think it's enough here to expose some problems

Reply:
IMO, nothing is immutable, you can introduce more flags for the new tests.

Reply:
Can we control the hit rate? Or is it stable?

Reply:
It depends on `generate_string`, I think it's a very low rate currently. We can reuse the seed which is used for random to control the rate.

Reply:
okay~